Improve doc note about configuration from brew (#32659)
grafana/grafana: docs/sources/administration/configuration.md

### macOS

> By default, the configuration file is located at `/usr/local/etc/grafana/grafana.ini`. If grafana was installed with Homebrew, you will need to edit `grafana.ini` directly. Otherwise, add a configuration file named `custom.ini` to the `conf` folder to override any of the settings defined in `conf/defaults.ini`.

Reviewer: ```suggestion
By default, the configuration file is located at `/usr/local/etc/grafana/grafana.ini`. For a Grafana instance that was installed with Homebrew, edit the `grafana.ini` file directly. Otherwise, add a configuration file named `custom.ini` to the `conf` folder to override any of the settings defined in `conf/defaults.ini`.
```
